docs/parameters/help_text.md

```diff
@@ -0,0 +1,76 @@
+## `paramsHelp()`
+
+This function returns a help message with the command to run a pipeline and the available parameters.
+Pass it to `log.info` to print in the terminal.
+
+It accepts two arguments:
+
+1. An example command, typically used to run the pipeline, to be included in the help string
+2. The file name of a Nextflow Schema file (Default: `nextflow_schema.json`)
+
+!!! Note
+
+    `paramsHelp()` doesn't stop pipeline execution after running.
+    You must add this into your pipeline code if it's the desired functionality.
+
+Typical usage:
+
+=== "main.nf"
+
+    ```groovy
+    --8<-- "examples/paramsHelp/pipeline/main.nf"
+    ```
+
+=== "nextflow.config"
+
+    ```groovy
+    --8<-- "examples/paramsHelp/pipeline/nextflow.config"
+    ```
+
+=== "nextflow_schema.json"
+
+    ```json
+    --8<-- "examples/paramsHelp/pipeline/nextflow_schema.json"
+    ```
+
+Output:
+
+```
+--8<-- "examples/paramsHelp/log.txt"
+```
+
+!!! warning
+
+    We shouldn't be using `exit` as it kills the Nextflow head job in a way that is difficult to handle by systems that may be running it externally, but at the time of writing there is no good alternative.
+    See [`nextflow-io/nextflow#3984`](https://github.com/nextflow-io/nextflow/issues/3984).
+
+## Specific parameter help
+
+By default, when printing the help message only a selection of attributes are printed: type of the variable, accepted options (enums), description and default value.
+
+To print the complete information for a single parameter, pass the parameter name to the `--help` option.
+
+For example, with the example above:
+
+```bash
+nextflow run my_pipeline --help outdir
```

Comment:
does this require any specific setting? im trying this with rnaseq and it still prints the whole `--help` instead of any specific parameter like outdir/genome/aligner

Reply:
Shouldn't do, no - that's the exact command I ran to get the output.

Reply:
i ll test again after stable release :)